docs/writing-plugins.md

```diff
@@ -303,65 +303,65 @@ the event, for example the following could be defined in `info.js`:
+    it must be unique across all plugins, and it's recommended that it is
     of format `plugin_name:message_name`.  The suffix `_event` will be
-    appened to the type.
+    appended to the type.
 
 **links**:
     An array of strings describing which links this event can be sent
     over.  Direction matters, `'master-slave'` means the event can be
-    sent from the master to slave, but can't be sent back the other way
+    sent from the master to the slave, but can't be sent back the other way,
     unless `'slave-master'` is also present in the links array.
 
 **forwardTo**:
-    Target to forward event to.  Can be either `'master'` to indicate a
-    slave should forward it to the master server or `'instance'` to
+    Target to forward an event to.  Can either be `'master'`, to indicate a
+    slave should forward it to the master server, or `'instance'`, to
     indicate it should be forwarded to the instances specified by the
     `instance_id` event property.  This works by using a default handler
     for the event at the links that forward it.
 
 **broadcastTo**:
-    Target to broadcast this message towards.  Currently only
+    Target to broadcast this message towards.  Currently, only
     `'instance'` is supported and means the event will be broadcast to
     all instances downstream of the target it's sent to, but not back
     from where it came from.  This means that sending an event to a
     slave from an instance will cause it to be broadcast to all
-    instances of that slave except for the instance it came from.
+    instances of that slave, except for the instance it came from.
 
 **eventProperties**:
     Object with properties mapping to a JSON schema of that property
     that specifies what's valid to send in the event.  This is
-    equivalent to using the `properties` keyword in JSON schema except
+    equivalent to using the `properties` keyword in JSON schema, except
     that the properties specified are implicitly required and additional
     properties are not allowed.  See [this guide][guide] for an
-    introduction to writing JSON schemas
+    introduction to writing JSON schemas.
 
 The forwardTo and broadcastTo can be combined such that specifying
 `'master'` as the forwardTo value and `'instance'` as the broadcastTo
 value will cause the event to be broadcast to all instances in the
-cluster.  For this to work you will need to specify instance-slave,
-slave-master and master-slave, slave-instance as the links.
+cluster.  For this to work, you will need to specify `instance-slave`,
+`slave-master`, `master-slave`, and `slave-instance` as the links.
 
 Keep in mind when forwarding events that if the target an event is being
-forwarded to is not online the event will be dropped.  If you need a
-confirmation that the message was received use a request.
+forwarded to is not online, the event will be dropped.  If you need a
+confirmation that the message was received, use a request.
```

Comment:
I suggest rewording the second sentence to "Use a request if you need a confirmation that the message was received."